### PR TITLE
[DC-1269] Remove references to integration

### PR DESCRIPTION
**Pull request by s-rubenstein**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/dc-1269

## Addresses
This removes all references to the integration environment. It is a little tricky to detangle from the integration tests, this PR probably warrants a more careful look.

### CODE REVIEW

File: docs/infrastructure.md [hunk 1]
```diff
@@ -66,18 +66,17 @@ Some actions can be defined within a yaml passing bash to it or it can be abstra
...
   - terraform lint/format
   - simple logic test
-  - Runs plan against current dev/integration  environment
+  - Runs plan against current dev environment
 - [Terraform PR merge](https://github.com/broadinstitute/terraform-jade/blob/master/.github/workflows/terraformMerge.yml)
   - On merge to master
   - terraform lint/format
   - simple logic test
-  - Applies plan against current dev/integration environment
+  - Applies plan against current dev environment
```
Comment (snf2ye) [inline docs/infrastructure.md]: We can remove this whole chunk since it references the terraform-jade repo

**Review by snf2ye — APPROVED** [review]

File: datarepo-clienttests/src/main/java/common/commands/PrintHelp.java [hunk 2]
```diff
@@ -115,7 +115,7 @@ public static void printHelp() throws IOException {
             + "Example Workflow (7): Execute a test configuration while locking the server, collect the measurements generated by the server during the run, and upload the results"
             + ANSI_RESET
             + System.lineSeparator()
-            + "export TEST_RUNNER_SERVER_SPECIFICATION_FILE=\"mmdev.json\"\n"
+            + "export TEST_RUNNER_SERVER_SPECIFICATION_FILE=\"localhost.json\"\n"
```
Comment (rjohanek) [inline datarepo-clienttests/src/main/java/common/commands/PrintHelp.java:118]: why these changes? what is mmdev

Comment (rjohanek) [reply datarepo-clienttests/src/main/java/common/commands/PrintHelp.java:118]: oh its a personal deployment? So this pr is not only removing references to integration but also to personal deployments, is that right?

**Review by rjohanek — APPROVED** [review]

File: datarepo-clienttests/src/main/java/common/commands/PrintHelp.java (hunk 2 above)
Comment (s-rubenstein) [reply datarepo-clienttests/src/main/java/common/commands/PrintHelp.java:118]: mmdev is an _old_ personal deployment for a Verily engineer

File: .github/dependabot.yml [hunk 3]
```diff
@@ -3,13 +3,6 @@ updates:
   # Enable version updates for Gradle
   - package-ecosystem: "gradle"
     directory: "/"
-    # See workflows/int-and-connected-test-run.yml for the current list of k8s namespaces available
-    # for integration testing.
-    # Presently, there are only 4, and these tests often take 2 hours to run.
-    # Keeping the open PR limit low at 4 so that concurrent Dependabot PR integration test runs all
-    # stand a chance at success rather than some being likely to time out waiting for an available
-    # namespace.
-    open-pull-requests-limit: 4
```
Comment (fboulnois) [inline .github/dependabot.yml:12]: 🎉

**Review by fboulnois — APPROVED** [review]
looks good 👍
